src/material/schematics/ng-update/upgrade-rules/hammer-gestures-v9/hammer-gestures-rule.ts

```diff
@@ -235,6 +250,45 @@ export class HammerGesturesRule extends MigrationRule<null> {
     });
   }
 
+  /** Removes all references to the "HammerModule" from "@angular/platform-browser". */
+  private _removeHammerModuleReferences() {
```

Comment:
@jelbourn As I've shortly explained in the PR description, it will be a rare scenario that we need to remove the `HammerModule`, unless you can think of a different scenario?

if you think we should just not handle this case, I'm good with it. otherwise, it will be just that function that handles that scenario.